hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java

```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.config;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+
+public class MapAttributeConfigReadOnlyTest {
```

Comment:
That's a good one! I should start drinking coffee before doing reviews in the morning :)